docs_src/microservices/application/AppServiceConfigurable.md

```diff
@@ -40,6 +41,53 @@ Next, each function and its required information is listed. Each function typica
 
 That's it! Now we can run/deploy this service and the functions pipeline will process the data with functions we've defined.
 
+## Pipeline Per Topic
+
+!!! edgey "EdgeX 2.1"
+    Pipeline Per Topic is new for EdgeX 2.1
+
+The above pipeline configuration in [Getting Started](#getting-started) section is the preferred way if your use case only requires a single functions pipeline. For use cases that require multiple functions pipelines in order to process the data differently based on the `profile`, `device` or `source` for the Event, there is the Pipeline Per Topic feature. This feature allows multiple pipelines to be configured in the `[Writable.Pipeline.PerTopicPipelines]`section. This section is a map of pipelines. The map key must be unique , but is used so can be any value. Each pipleline is defined by the following configuration settings:
+
+- Id - This is the unique ID given to each pipeline
+- Topic - This is the MessageBus topic that controls when the pipeline is executed. See [Pipeline Per Topic](../AdvancedTopics/#pipeline-per-topic)  for details on using wildcards in the topic.
+- ExecutionOrder - This is the list of functions, in order, that the pipeline will execute. Same as `ExecutionOrder` in the above example in the  [Getting Started](#getting-started) section
+
+!!! example "Example - Writable.Pipeline.PerTopicPipelines"
+    In this example Events from the device  `Random-Float-Device` are transformed to JSON and then HTTP exported. While Events for the source `Int8` are transformed to XML and then HTTP exported to same endpoint. Note the custom naming for `TransformJson` and `TransformXml`. This is taking advantage of the [Multiple Instances of a Function](#multiple-instances-of-a-function) described below.
```

Comment:
As is, it is a phrase, not sure if this would have the same meaning, but you need a prepositional phrase or a different transition phrase to make your sentence complete.

```suggestion
    In this example Events from the device  `Random-Float-Device` are transformed to JSON and then HTTP exported. At the same time, Events for the source `Int8` are transformed to XML and then HTTP exported to same endpoint. Note the custom naming for `TransformJson` and `TransformXml`. This is taking advantage of the [Multiple Instances of a Function](#multiple-instances-of-a-function) described below.
```

Reply:
Updated